### DISCUSSION TITLE
Strange error when trying to execute notify automations.

### DISCUSSION THREAD

**Discussion by Dvalin21** [blakeblackshear/frigate#5709] [General]
I have everything setup, I go to test the notification and  nothing happens.

I check logs and this is what I get

Logger: homeassistant.components.automation.ha_storage_room_detection
Source: components/automation/__init__.py:254
Integration: Automation (documentation, issues)
First occurred: 12:40:48 AM (1 occurrences)
Last logged: 12:40:48 AM

Error rendering variables: ValueError: Template error: int got invalid input 'unknown' when rendering template '{{ fps_value|int if is_number(fps_value) or 5 }}' but no default was specified

Now I had this working previously on 12 beta9, but both frigate and homeassistant was on the same server.   Now I have Frigate on a different server with different ip. This shouldn't be a problem as I know Matt is used and I have it setup in the config.yml and on the server. Anyone has any idea why I would get this? Thanks

**Comment by blakeblackshear** [comment]
Stats are polled via http,so be sure to have port 5000 open.

**Comment by Dvalin21** [comment]
It should be. Im now wondering if the conflict is with port 5000 on my
frigate server and port 5000 on my nas where my HAOS is hosted

**Reply by blakeblackshear** [reply]
You should see the /api/stats request in the nginx logs from HomeAssistant in the UI. Something like this

```
2023-03-13 00:18:35.228356572  10.42.0.1 - - [12/Mar/2023:19:18:35 -0500] "GET /api/stats HTTP/1.1" 200 2791 "-" "HomeAssistant/2023.2.5 aiohttp/3.8.1 Python/3.10" "-"
```

**Reply by Dvalin21** [reply]
> You should see the /api/stats request in the nginx logs from HomeAssistant in the UI. Something like this
> 
> ```
> 2023-03-13 00:18:35.228356572  10.42.0.1 - - [12/Mar/2023:19:18:35 -0500] "GET /api/stats HTTP/1.1" 200 2791 "-" "HomeAssistant/2023.2.5 aiohttp/3.8.1 Python/3.10" "-"
> ```

Thanks, I tried changing the port for frigate on the host machine its on, however I haven't had much change. 

I'm still getting the following refused to connect

134#134: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.30, server: , request: "GET /api/stats HTTP/1.1", upstream: "http://127.0.0.1:5001/stats", host: "10.0.0.28:5003"

What is the "http://127.0.0.1:5001/stats, the only thing that seems to be causing the connection issue.

**Reply by Dvalin21** [reply]
Update: Switched the port to 5001 and no longer getting the refuse connection errors.
